android-sdk/tasks/main.yml

```diff
@@ -75,15 +83,15 @@
   when: accept is undefined
 
 - pause:
-    prompt:  "{{ output.stdout }} Hit enter to accept the Licence or hit Ctrl _c and 'A' to reject and cancel the installation "
+    prompt:  "{{ rsh_cmd.stdout.split('--------------------')[1] }} Hit enter to accept the Licence or hit Ctrl _c and 'A' to reject and cancel the installation. ** Note ** This task may take some time, please be patient "
   when: accept is defined and accept
 
 - name: "Accepting the licence"
```

Comment:
Can we add 'this may take some time' here. other than that it worked for me so can go ahead and merge